day_6/exercises/person.js

```diff
@@ -7,5 +7,22 @@ Create a person class with at least 2 properties and 2 behaviors.  Call all
 person methods below the class so that they print their result to the
 terminal.
 */
+class Person {
+  constructor (hair, eyes) {
+    this.hair = hair;
+    this.height = height;
+  }
+  dyeHair (newColor) {
+    this.hair = newColor
+    console.log(this.hair)
+  }
+  wearHeels(heelSize) {
+    this.height = heelSize + this.height
+    console.log(this.height)
+  }
+};
 
-// YOUR CODE HERE
+var jessica = new Person("red","5 feet" )
+
+person.dyeHair("brown")
+person.wearHeels("6 inch")
```

Comment:
Great job! After these method calls, you probably want to `console.log(jessica)` to verify this object now has brown hair and is taller!